.github/workflows/testing.yaml

```diff
@@ -0,0 +1,248 @@
+name: testing
+
+on: [push, pull_request]
+
+jobs:
+  testing_linux:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - release/1.10.7
+          - release/1.10.8
+          - release/1.10.9
+          - release/1.10.10
+          - release/2.4.1
+          - release/2.4.2
+          - release/2.4.3
+#          - release/2.5.1 # there is a crash in the cartridge helper
+          - release/2.5.2
+          - release/2.5.3
+          - release/2.6.1
+          - release/2.6.2
+          - release/2.6.3
+          - release/2.7.1
+          - release/2.7.2
+          - release/2.8.1
+          - live/1.10
+          - live/2.6
+          - live/2.7
+          - live/2.8
+          - live/2.9
+        # Add an extra check or modify a system environment
+        # on one job.
+        include:
+          # Applicable for: 2.6.3, 2.7.2+, 2.8.1+, 2.9+ (gh-4559).
+          #
+          # Verify that system's libcurl headers are not required.
+          - tarantool: live/2.8
+            dont_install_system_libcurl_header: true
+          # Applicable for: 2.6.3, 2.7.2+, 2.8.1+, 2.9+ (gh-4559).
+          #
+          # Verify that tarantool's libcurl headers are preferred
+          # when the system provides them too.
+          - tarantool: live/2.9
+            break_system_libcurl_header: true
```

Comment:
Do you ever read the code you copied? If so, explain how libcurl is relevant here. If not, please, never do attempt to proceed with a patch this way again.

Reply:
My mistake is that I did not mark this PR as draft / WIP

Reply:
moved it to another PR